src/app/api/api_v1/endpoints/test.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, Body, HTTPException, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from app.api import deps
+
+router = APIRouter()
+
+@router.get("/test/{id}")
```

Comment:
We should have two newlines around this top level function according to [PEP8](https://peps.python.org/pep-0008/#blank-lines).